devtools/conda-envs/test_env.yaml

```diff
@@ -1,37 +1,19 @@
 name: openmmforcefields-test
-
 channels:
   - conda-forge
-
 dependencies:
-    # Base depends
   - python
   - pip
-
-    # Testing
   - pytest
   - pytest-cov
   - pytest-xdist
   - pytest-randomly
-
-    # Requirements for converted force field installer
-  - openmm >=7.6.0
-
+  - openmm >=8.1.2
   - openff-toolkit >=0.11
-
-    # Requirements for conversion tools
   - pyyaml
   - ambertools >=22,<24
   - lxml
   - networkx
-
-    # JSON caching of residue templates
   - tinydb
-
-    # Validating URLs
   - validators
-
-    #
-    # Espaloma requirements below
-    #
-  # espaloma >=0.3.1
+  - rdkit !=2024.03.4
```

Comment:
Nice catch

Reply:
looks like it is working for some versions of osx! it was random, Irfan and I were checking out an openff repo and saw a PR that mentioned issues with rdkit

Reply:
@lilyminium [suggests](https://github.com/rdkit/rdkit/issues/7619#issuecomment-2237584556) that it may be a problem with a specific build of rdkit 2024.03.4 - Build 0 is OK, but build 1, [migrating to numpy 2](https://github.com/conda-forge/rdkit-feedstock/pull/160), seems to be the problem.

Reply:
That is good to know, I am not sure the syntax to skip a particular build without knowing the full build string, but it would be nice to add that to the conda-forge recipe for this release

Reply:
The broken builds have been pulled and I'm unpinning this